### DISCUSSION TITLE
Color scheme in Demo

### DISCUSSION THREAD

**Discussion by chrisgrieser** [VoxelPrismatic/rabbit.nvim#2] [Q&A] [answered]
Hi, may I ask what the color scheme is you are using in the demo?

**Comment by VoxelPrismatic** [comment, marked as answer]
I am using [RosePine Dawn](https://github.com/rose-pine/neovim), but I have made [my own fork](https://github.com/voxelprismatic/rosepine-neovim) and added a Sakura swatch. Now looks like this:
![sakura pine dawn](https://cdn.discordapp.com/attachments/1092543067191525407/1247733666289811486/image-15.png?ex=66611a07&is=665fc887&hm=79545cd151127a0eb903c12ec01cf3a1b60d634458f5369652cd2d07e9cbc3f0&)

**Reply by chrisgrieser** [reply]
Thank you!
